### ISSUE TITLE
test_network.py:test_gre_profile failure due to KeyError

### ISSUE THREAD

**Issue by ssorenso** [F5Networks/f5-openstack-agent#858]
* Title: test_network.py:test_gre_profile failure due to KeyError
* Attachments: 
* Details: 

```

setup = <class 'test_network.ExecTestEnv'>

    @log_test_call
    def test_gre_profile(setup):
        te = setup
        name = 'test_gre_profile'
        profile = network_helper.create_l2gre_multipoint_profile(te.mgmt_root, name,
                                                                 te.partition)
        add_resource_teardown(te.request, profile)
        params = {'params': {'$filter': 'partition eq %s' % te.partition}}
        gc = te.mgmt_root.tm.net.tunnels.gres
        profiles = gc.get_collection(requests_params=params)
        profile_names = (r.name for r in profiles)
        assert(name in profile_names)
        p = te.mgmt_root.tm.net.tunnels.gres.gre
        p.load(name=name, partition=te.partition)
        payload = NetworkHelper.l2gre_multipoint_profile_defaults
        for k in payload.keys():
            if k == 'partition':
                continue
>           assert(p.__dict__[k] == profile.__dict__[k])
E           KeyError: 'name'

test_network.py:104: KeyError
```

Tests:
* `openstack_agent_liberty_12.1.1-overcloud.92.consoleText`
  * Type: failures
  * Test: `test_gre_profile`
  * sum: 0d197b46e07ca2c992c7add1287f0807

#### Agent Version
liberty - 96d5250fefec71d3cfc6da3a4a5d977e179ed037

#### Operating System
Nightly

#### OpenStack Release
Liberty

#### Bug Severity 
Low

#### Deployment
overcloud

**Comment by ssorenso** [comment]
This may be related (traceback looks similar to) #845

**Comment by ssorenso** [comment]
Additional traceback sum:
md5sum:4b936922df8bd97505ba7138387bc61c

**Comment by ssorenso** [comment]
Newton looks pretty similar, and a repeat in mitaka:
* `openstack_agent_mitaka_12.1.2-overcloud.6.consoleText`
  * Type: failures
  * Test: `test_gre_profile`
  * Sum: `md5sum:0c14a96d22c997f4370a6155ac5af8e8`
* `openstack_agent_newton_12.1.2-overcloud.6.consoleText`
  * Type: failures
  * Test: `test_gre_profile`
  * Sum: `md5sum:c68e49eee0cc34b713ecbd1f4feabf40`

Newton's error:
```
setup = <class 'test_network.ExecTestEnv'>

    @log_test_call
    def test_gre_profile(setup):
        te = setup
        name = 'test_gre_profile'
        profile = network_helper.create_l2gre_multipoint_profile(te.mgmt_root, name,
                                                                 te.partition)
        add_resource_teardown(te.request, profile)
        params = {'params': {'$filter': 'partition eq %s' % te.partition}}
        gc = te.mgmt_root.tm.net.tunnels.gres
        profiles = gc.get_collection(requests_params=params)
        profile_names = (r.name for r in profiles)
        assert(name in profile_names)
        p = te.mgmt_root.tm.net.tunnels.gres.gre
        p.load(name=name, partition=te.partition)
        payload = NetworkHelper.l2gre_multipoint_profile_defaults
        for k in payload.keys():
            if k == 'partition':
                continue
>           assert(p.__dict__[k] == profile.__dict__[k])
E           KeyError: 'defaultsFrom'

test_network.py:104: KeyError
```

**Comment by richbrowne** [comment]
Fixed in 0cdebc5be80dc4ba8759672f6ee808123c590e5e
